pkg/controller/workspace/workspace_controller.go

```diff
@@ -176,6 +176,11 @@ func (r *ReconcileWorkspace) Reconcile(request reconcile.Request) (reconcileResu
 		return reconcile.Result{}, err
 	}
 
+	if workspace.DeletionTimestamp != nil {
+		reqLogger.V(5).Info("Skipping reconcile of deleted resource")
```

Comment:
Why 5 but not 6 or 7?)
For we that int log levels are still not clear )

I tried to introduce some constants for typical levels - info, debug, trace but I did not manage to finalize it.
It probably should be handled in the scope of https://github.com/eclipse/che/issues/16659

Reply:
It's pretty arbitrary, I agree.